.github/workflows/build_and_test.yml

```diff
@@ -1116,7 +1116,10 @@ jobs:
 
             gh workflow --repo neondatabase/infra run deploy-proxy-prod.yml --ref main \
               -f deployPgSniRouter=true \
-              -f deployProxy=true \
```

Comment:
we're missing base proxy-scram.

Reply:
we have `deployLegacyProxyScram`, is there one more proxy scram than this?

Reply:
waiting on:

previously deployProxyScram was pointed to deploy_legacy_scram_proxy I don't see only deploy_scram_proxy (without legacy)

Reply:
> we're missing base proxy-scram.

It seems this part is missed in `_deploy-proxy.yml` workflow, and the base scram proxy always gets deployed: https://github.com/neondatabase/infra/blob/8db8b5b2342beda3526f8aa0338ecbb1fd4a9904/.github/workflows/_deploy-proxy.yml#L189-L197 (there's no if-condition)

Reply:
going to add deployProxyScram https://github.com/neondatabase/infra/pull/2152

Reply:
@bayandin  Yes, added in base CI as well.